libraries/RW/Prometheus/Prometheus.py

```diff
@@ -114,6 +113,7 @@ def query_instant(
         Return Value:
         |   prometheus_response: dict  |
         """
+        point_in_time=datetime.now()
```

Comment:
I'd still like to see `point_in_time` be an optional param in the event users would like to pull a past data point.

So add:
`point_in_time = None,` to the function signature (line 105)

then a conditional to default it as you did on line 116
```
if point_in_time == None:
    point_in_time=datetime.now()
```

Reply:
good point, changes made